charts/fleet/templates/deployment.yaml

```diff
@@ -56,13 +56,17 @@ spec:
           allowPrivilegeEscalation: false
           readOnlyRootFilesystem: true
         {{- end }}
-        {{- if .Values.cpuPprof }}
         volumeMounts:
+          - mountPath: /tmp
+            name: tmp
+        {{- if .Values.cpuPprof }}
```

Comment:
I think this is obsolete now, /tmp is already a volume. As far as I can tell `/tmp/pprof` is fine. @moio ?

Reply:
Ah thanks. I didn't expand the diff :) 👍